--- numpy-1.14.6.orig/numpy/random/tests/test_random.py	2018-09-23 10:21:50.000000000 -0400
+++ numpy-1.14.6/numpy/random/tests/test_random.py	2019-11-02 13:46:02.652236876 -0400
@@ -591,7 +591,7 @@
 
     def test_hypergeometric(self):
         np.random.seed(self.seed)
-        actual = np.random.hypergeometric(10.1, 5.5, 14, size=(3, 2))
+        actual = np.random.hypergeometric(10, 5, 14, size=(3, 2))
         desired = np.array([[10, 10],
                             [10, 10],
                             [9, 9]])
@@ -896,6 +896,8 @@
             def __int__(self):
                 raise TypeError
 
+            __index__ = __int__
+
         throwing_int = np.array(1).view(ThrowingInteger)
         assert_raises(TypeError, np.random.hypergeometric, throwing_int, 1, 1)
 
