diff --git a/numpy/distutils/system_info.py b/numpy/distutils/system_info.py
index 683b15d..01d547d 100644
--- a/numpy/distutils/system_info.py
+++ b/numpy/distutils/system_info.py
@@ -211,6 +211,34 @@ if sys.platform == 'win32':
     default_src_dirs = ['.']
     default_x11_lib_dirs = []
     default_x11_include_dirs = []
+    vcpkg_include_dirs = [
+        'include',
+        'include/suitesparse',
+    ]
+    vcpkg_lib_dirs = [
+        'lib',
+    ]
+    if sys.version_info >= (3, 3):
+        # VCpkg is the de-facto package manager on windows for C/C++
+        # libraries. If it is on the PATH, then we append its paths here.
+        # We also don't re-implement shutil.which for Python 2.7 because
+        # vcpkg doesn't support MSVC 2008.
+        vcpkg = shutil.which('vcpkg')
+        if vcpkg:
+            vcpkg_dir = os.path.dirname(vcpkg)
+            if platform.architecture() == '32bit':
+                specifier = 'x86'
+            else:
+                specifier = 'x64'
+            vcpkg_root = os.path.join(
+                vcpkg_dir, 'installed', specifier + '-windows')
+
+            default_lib_dirs.extend(
+                os.path.join(
+                    vcpkg_root, d.replace('/', os.sep)) for d in vcpkg_lib_dirs)
+            default_include_dirs.extend(
+                os.path.join(
+                    vcpkg_root, d.replace('/', os.sep)) for d in vcpkg_include_dirs)
 else:
     default_lib_dirs = libpaths(['/usr/local/lib', '/opt/lib', '/usr/lib',
                                  '/opt/local/lib', '/sw/lib'], platform_bits)
