diff --git a/numpy/core/tests/test_mem_overlap.py b/numpy/core/tests/test_mem_overlap.py
index 3c8e0e722..ca9ab9b89 100644
--- a/numpy/core/tests/test_mem_overlap.py
+++ b/numpy/core/tests/test_mem_overlap.py
@@ -375,6 +375,7 @@ def check_may_share_memory_easy_fuzz(get_max_work, same_steps, min_count):
                 infeasible += 1
 
 
+@pytest.mark.skipif(True, reason='Times out on qemu+aarch')
 @pytest.mark.slow
 def test_may_share_memory_easy_fuzz():
     # Check that overlap problems with common strides are always
